lib/filewatch/tail.rb

```diff
@@ -20,6 +26,13 @@ class Tail
 
     class NoSinceDBPathGiven < StandardError; end
 
+    alias_method :subscribe, :yield_subscribe
+
+    def self.new_observing(opts = {})
+      alias_method :subscribe, :observe_subscribe
```

Comment:
This feels... unsafe? Modifying the instance methods (of all instances?) Did you mean to use define_singleton_method ?